packages/react-aria-components/src/Tooltip.tsx

```diff
@@ -137,7 +137,8 @@ function TooltipInner(props: TooltipProps & {isExiting: boolean, tooltipRef: Ref
     crossOffset: props.crossOffset,
     isOpen: state.isOpen,
     arrowSize: arrowWidth,
-    arrowBoundaryOffset: props.arrowBoundaryOffset
+    arrowBoundaryOffset: props.arrowBoundaryOffset,
+    shouldFlip: props.shouldFlip
```

Comment:
we do this in RSP implementation, so will eventually need it here. Thanks for adding it.